### PR TITLE
Chore/add scaffolding for scss

### PR DESCRIPTION
**Pull request by alexanderleegs**
This PR adds support for sass and common sass variables with our existing template. Dummy values are there to demonstrate functionality - will be removed in a subsequent PR.

### CODE REVIEW

File: assets/scss/styles.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+---
```
Comment (alexanderleegs) [inline assets/scss/styles.scss:2]: Jekyll only converts sass files if the .scss starts with two lines of triple dashes (to skip the front matter), so this is required to generate `scss/styles.css`

**Review by harishv7 — APPROVED** [review]

File: assets/scss/styles.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+@import "homepage/text-cards"
```
Comment (seaerchin) [inline assets/scss/styles.scss]: should be `@use` cos this is `.scss` no? see [here](https://sass-lang.com/documentation/at-rules/use/)

Comment (alexanderleegs) [reply assets/scss/styles.scss]: swapped in https://github.com/isomerpages/isomerpages-template/pull/303/commits/fa9db7f89dcd0e3a4d6d00277317e44c285aa042!

Comment (seaerchin) [reply assets/scss/styles.scss]: Lgtm lfg

**Pull request merged**